lightclient-circuits/src/committee_update_circuit.rs

```diff
@@ -189,29 +195,20 @@ impl<S: Spec, F: Field> CommitteeUpdateCircuit<S, F> {
                 .expect("bad bls12_381::Fq encoding")
         });
 
-        let poseidon_commitment = poseidon_hash_fq_array::<bn256::Fr>(pubkeys_x, limb_bits);
-
-        let mut pk_vector: Vector<Vector<u8, 48>, { S::SYNC_COMMITTEE_SIZE }> = args
-            .pubkeys_compressed
-            .as_slice()
-            .iter()
-            .map(|v| v.as_slice().try_into().unwrap())
-            .collect_vec()
-            .try_into()
-            .unwrap();
-
-        let ssz_root = pk_vector.hash_tree_root().unwrap();
+        let poseidon_commitment =
+            poseidon_hash_fq_array::<bn256::Fr>(pubkeys_x, limb_bits - (limb_bits % 2));
 
         let finalized_header_root = args.finalized_header.clone().hash_tree_root().unwrap();
 
+        let finalized_header_root_hilo = {
+            let bytes = finalized_header_root.as_ref();
+            let hash_lo = u128::from_le_bytes(bytes[16..].try_into().unwrap());
+            let hash_hi = u128::from_le_bytes(bytes[..16].try_into().unwrap());
+            [hash_lo, hash_hi].map(bn256::Fr::from_u128)
+        };
+
         let instance_vec = iter::once(poseidon_commitment)
-            .chain(ssz_root.as_ref().iter().map(|b| bn256::Fr::from(*b as u64)))
```

Comment:
I might be missing something. Why dont we need the SSZ root anymore?

Reply:
it was redundant, see tg chat with Veridise